.pre-commit-hooks.yaml

```
@@ -1,7 +1,9 @@
 - id: sphinx-lint
-  name: Sphinx lint
+  name: Sphinx Lint
   description: 'Searches for common typos in sphinx-flavored rst files.'
```

```suggestion
  description: 'Searches for common problems in Sphinx-flavored reST files'
```
Or perhaps reword some other way?

---

This rewording looks good to me!